src/null-safety/index.md

```diff
@@ -78,8 +78,8 @@ Dart null safety support is based on the following three core design principles:
 
 ## Dart 3 and null safety
 
-Dart 3---planned for a mid-2023 release---always has sound null safety.
-Dart 3 will prevent code from running without it.
+Dart 3 always has sound null safety,
+and will prevent code from running without it.
```

Comment:
If it doesn't change the meaning, I would make this more active:

```suggestion
Dart 3 is fully null safe. 
```

The second clause, "... will prevent code from running without it", seems like it's saying the same thing as the next line, "Packages developed without...".

I get rid of that clause and just keep line 84 because it's more specific, and maybe add "Dart 3" to it:

> Packages developed in Dart 3 without null safety support will cause issues when resolving dependencies:

Reply:
I think that changes the meaning potentially (it sounds like Dart 3 was the release where we made the core libraries null safety or something like that...).

Riffing on another comment you had elsewhere in the review, could we say:

```
Dart 3 has sound null safety built-in,
and will prevent code from running without it.
```